src/@types/ipfs-http-client/index.d.ts

```diff
@@ -12,7 +12,7 @@ declare module 'ipfs-http-client' {
 
   interface Options {
```

Comment:
This updated version of the `ipfs-http-client` already has bundled TS typings, so this folder should be completely removed. You will need to do some tweaks as well, you can see the Pinner's PR for details...

Reply:
Yes, I try it. But I can not find the corresponding types for `IpfsObject, IpfsResult, RegularFiles`.

Reply:
Yeah, you have to dig for them around a bit. Lets wait a bit, because I am working on this in the https://github.com/rsksmart/rif-storage-js so you can then see where I dig them out ;-)

Reply:
Cool. Also, it's kind of strange how you import these types `ipfs-http-client/src/lib/core` which point me to `JS` file. Instead all typing placed in another package `ipfs-core-types`

Reply:
So, maybe we can merge this one. And I will create an issue for `IPFS typing` and fix it in separated PR

Reply:
Ok, sounds good.